chemper/graphs/cluster_graph.py

```diff
@@ -100,6 +105,36 @@ def add_atom(self, atom):
             """
             self.decorators.add(self.make_atom_decorators(atom))
 
+        def compare_atom(self, atom):
+            """
+
+            Parameters
+            ----------
+            atom: chemper Atom
+
+            Returns
+            -------
+            score: int
+                 how similar is atom to current storage, max of 7 for all decorators identical
```

Comment:
"How similar is atom to current storage" is opaque to me. Can you clarify? Does this mean basically out of the total number of possible decorators (7?) how many of these are the same? (And it is only applied when the element is already the same?)

Reply:
Yes, your intuition was correct, but I will update the wording so that is more clear.